db/seeds.rb

```diff
@@ -1,7 +1,33 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+class Seeds
+	def create
+		generate_users
+		generate_specific_users
+	end
+
+	def generate_users
+		20.times do
+      user = User.create(
+        user_name: Faker::Name.first_name,
+        provider: "Facebook",
+        email: Faker::Internet.email,
+        password: "password"
+      )
+    end
+    p 'Users Created'
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not write to stdout. Use Rails' logger if you want to log.